packages/react-native/React/CoreModules/RCTDevLoadingView.mm

```diff
@@ -220,7 +220,9 @@ - (void)showMessage:(NSString *)message color:(RCTUIColor *)color backgroundColo
           self->_hiding = false;
         }];
 #else // [macOS]
-    [RCTKeyWindow() endSheet:self->_window];
+    for (NSWindow *window in [RCTKeyWindow() sheets]) {
+      [RCTKeyWindow() endSheet:window];
+    }
```

Comment:
This would also dismiss any sheet not presented by devLoadingView, so maybe we should add an identifier or ensure we only ever have one sheet from devLoadingView

Reply:
This should be fixed